File: demo_core_av/app/src/main/java/com/bandyer/demo_core_av/room/utils/ScreenSharingUtils.java
package com.bandyer.demo_core_av.room.utils;

import android.content.Context;
import android.content.Intent;

import static com.bandyer.demo_core_av.room.utils.ScreenShareForegroundService.ACTION_START_FOREGROUND_SERVICE;

/**
 * @author kristiyan
 */
public class ScreenSharingUtils {

    public static void showScreenShareNotification(Context context) {
        Intent intent = new Intent(context, ScreenShareForegroundService.class);
        intent.setAction(ACTION_START_FOREGROUND_SERVICE);
        context.startService(intent);

    }

    public static void hideScreenShareNotification() {
        if (ScreenShareForegroundService.getInstance() == null) return;
        ScreenShareForegroundService.getInstance().dispose();
    }

}

File: demo_core_av/app/src/main/java/com/bandyer/demo_core_av/room/utils/ScreenSharingUtils.kt
package com.bandyer.demo_core_av.room.utils

import android.content.Context
import android.content.Intent

/**
 * @author kristiyan
 */
object ScreenSharingUtils {
    fun showScreenShareNotification(context: Context) {
        val intent = Intent(context, ScreenShareForegroundService::class.java)
        intent.action = ScreenShareForegroundService.ACTION_START_FOREGROUND_SERVICE
        context.startService(intent)
    }

    fun hideScreenShareNotification() {
        ScreenShareForegroundService.instance?.dispose()
    }
}